### DISCUSSION TITLE
PRG

### DISCUSSION THREAD

**Discussion by Marc91360** [frntc/SIDKick-pico#84] [General]
I'm probably a little  too curious, but which PRGs have you chosen to join / which ones would you recommend?

Best regards,

Marc

**Comment by frntc** [comment]
Not sure if you're asking me :-) I mainly used the feature during development, so it has PRGs I wanted to test more often (if not using the Sidekick64). In "real-use" I'd maybe put fast loaders, utilities and such -- but my main use of C64s is working on my projects :-)

**Reply by banman11** [reply]
Brilliant concept to have .prg's residing on SIDKick pico.

**Comment by SX64man** [comment]
I added these 16 prg to help me test the various options:
SIDKick pico patch v0.2

reading SIDKick pico firmware.

parsing PRG list:
       menu name : filename
       SKPICOTIMING : PRG/SKpicotiming.prg
       SIDTESTER : PRG/sidtester.prg
       SIDTESTER-2SID : PRG/sidtester-2sid.prg
       SIDALYZER : PRG/sidalyzer.prg
       SIDBENCH : PRG/sidbench1.prg
       SIDKEYS : PRG/sidkeys.prg
       SIDPLAY64-SD2IEC : PRG/sidplay64-sd2iec.prg
       FM-YAMAHA-DEMO : PRG/fm_yam_demo_cq_xentax.prg
       CYNTHCART : PRG/cynthcart.prg
       GUILTY_PLEASURES : PRG/guilty_pleasures.prg
      PHOBOS-STEREO : PRG/Phobos_Stereo.prg
      PSEUDOTUNES : PRG/pseudotunes_PAL_NTSC.prg
      STEREOPHONIK-8580 : PRG/Stereophonik8580.prg
      STEREOPHONIK-6581 : PRG/Stereophonik6581.prg
      YETBIGGERBEATJMR : PRG/YetBiggerBeatJMR.prg
      DUBSTEP-PASSE : PRG/Dubstep Passe by Randall of Arise.prg

writing new SIDKick pico firmware to 'SKpicoPRG.uf2'.

**Comment by banman11** [comment]
👍

Good work.
